ui/raidboss/data/06-ew/raid/p1s.ts

```diff
@@ -0,0 +1,206 @@
+      netRegexDe: NetRegexes.startsUsing({ id: '65F8', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '65F8', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '65F8', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.outThenIn!(),
+      outputStrings: {
+        outThenIn: Outputs.outThenIn,
+      },
+    },
+    {
+      id: 'P1S Gaoler\'s Flail In => Out',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '65F9', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '65F9', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '65F9', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '65F9', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.inThenOut!(),
+      outputStrings: {
+        inThenOut: Outputs.inThenOut,
+      },
+    },
+    {
+      id: 'P1S Heavy Hand',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6629', source: 'Erichthonios' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6629', source: 'Erichthonios' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6629', source: 'Érichthonios' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6629', source: 'エリクトニオス' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'P1S Pitiless Flail of Grace',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '660E', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '660E', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '660E', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '660E', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.directions!(),
+      outputStrings: {
+        directions: {
+          en: 'Tankbuster+Knockback => Stack',
+        },
+      },
+    },
+    {
+      id: 'P1S Pitiless Flail of Purgation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '660E', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '660E', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '660E', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '660E', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.directions!(),
+      outputStrings: {
+        directions: {
+          en: 'Tankbuster+Knockback => Flare',
+        },
+      },
+    },
+    // Copy/paste from normal, seems to be the same
+    {
+      id: 'P1S Hot/Cold Spell',
```

Comment:
This mechanic works completely different from normal. For example you maybe don't want to get hit by the other color immediately.

Calling this out immediately gives a wrong idea about the mechanic I think.

Reply:
Handling this mechanic will likely boil down to cactbot either just calling out which of the three patterns were selected or following some common PF strategy.

For instance, instead of just the pattern, it could state where to move, role agnostic, to the common strategy. In all cases I see right now, there would be three callous. Call out two is basically middle column in all cases, but to specify which of the 3 rows requires strategy based on the player locations of the first set as to not screw the third set up. Call out one is something like clock positions or rotate clock position as the common ive seen strategy goes. Call out three is something like outer tile.

Reply:
My static does a really weird tank adjust strat for now so I'm curious to see how the strategy for this will be in pf.

Second explosion is not mid for all, north and south need to be soaked (or do I misunderstand you?).

Anyways I'd just remove it for now until there is proper handling.

Reply:
Second explosion is "middle column": the column meaning I set the floor into a 3x3 table and so there would be three possible tiles to choose from and up to the player to interpret where they should go for the second explosion.

Reply:
I'll add that i think it does not matter which middle column tile they end on so long as they don't double fire or double ice. I believe they can die to the 3rd one without causing a wipe so then it's not too bad if cactbot were to be even less specific about a strategy there.